libbeat/beat/beat.go

```diff
@@ -447,6 +454,48 @@ func (b *Beat) loadDashboards() error {
 	return nil
 }
 
+func (b *Beat) loadTemplate() error {
+	if *setup {
```

Comment:
It should be always loaded. The check on line 472 is different from the Dashboard one.